### PR TITLE
Update Fixtures with the removal of Transfer-Encoding

### PR DESCRIPTION
**Pull request by duduribeiro**
After https://github.com/dnsimple/dnsimple-developer/pull/337 we removed
the Transfer-Encoding from the fixtures.

This commit updates the project fixtures with the most recent ones.

### CODE REVIEW

File: src/dnsimple-test/Services/DomainsEmailForwardsTest.cs [hunk 1]
```diff
@@ -107,8 +107,8 @@ public void EmailForwardsData()
         }
 
         [Test]
-        [TestCase(1010, "228963",
-            "https://api.sandbox.dnsimple.com/v2/1010/domains/228963/email_forwards")]
+        [TestCase(1010, "235146",
```
Comment (ggalmazor) [inline src/dnsimple-test/Services/DomainsEmailForwardsTest.cs:110]: I'm just curious: are these changes required because the source fixture has changed or is there any other reason?

**Review by ggalmazor — APPROVED** [review]

**Review by DXTimer — APPROVED** [review]

**Review by san983 — APPROVED** [review]

File: src/dnsimple-test/Services/DomainsEmailForwardsTest.cs (hunk 1 above)
Comment (duduribeiro) [reply src/dnsimple-test/Services/DomainsEmailForwardsTest.cs:110]: @ggalmazor yeah. The content of the fixtures has been changed, so we need to update the tests as well
